src/dashboard/service.js

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+
+const NewsForumsService = {
```

Comment:
I'm not mad on this service name... `CommunityUpdatesService` maybe? Not a big fan of that either, but prefer it... Any other ideas?